interface/src/app/utils.spec.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Unit tests.
+ */
+
+// Test the hyperGeometricTest function
+describe('Test for hypergeometric test function in utils.js', function() {
+  beforeEach(module('adage.utils'));
+
+  var mockService;
```

Comment:
In this test, the name **mockService** is a little misleading, because it is assigned to the real "MathFuncts" service on line 13 (instead of being a MOCKED service). It may be better to simply call it **MathFuncts**.